brainstormforce/ultimate-addons-for-gutenberg
# Fix: Forms - Submit button width not applying.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/ae0bf28e-cbb1-4162-ae7f-5594e5abfa59?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes
- The issue is due to default is not set for border-color due to which it is setting undefined and when I checked it in the console it look like this 
`border: 2px solid undefined;`

### Screenshots

- https://www.awesomescreenshot.com/video/7061026?key=36b2153a252737b15e422ae782750ec0

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Forms
- Go to style > Submit > Select border style solid > set width and check it is applying or not. 

### Checklist:
